docs/UserGuide.md

```diff
@@ -500,21 +500,22 @@ The transferred save files can then be loaded readily when using this applicatio
-| **Delete an interview** | `delete_i <INDEX>` e.g.. `delete_i 1` | Deleted Interview: [Accountant [Bernice Yu, David Li] 2021-10-18 14:00 - 16:00 PENDING] |
-| **List all interviews** | `list_i` | Listed all interviews <br> 1. [Accountant [Bernice Yu, David Li] 2021-10-18 14:00 - 16:00 PENDING] <br> 2. [Project Manager [Bernice Yu] 2021-10-20 15:00 - 16:00 PENDING] |
-| **Find interview** | `find_i [position=POSITION]... [c=<Candidate Name>]... [date=DATE]... [time=TIME]... [duration=DURATION]... [interviewed=STATUS]...` <br> e.g find_i date=21/09/2021 time=1600 | Interviews found
-| **Unassign candidates** | `unassign i=<INTERVIEW_INDEX> c=<CANDIDATE_INDEX>...` e.g.. `unassign i=1 c=4`| Candidates removed from interview: [Project Manager [Bernice Yu] 2021-10-20 15:00 - 16:00 PENDING]: <br> 1. David Li |
-| **Assign candidates** | `assign i=<INTERVIEW_INDEX> c=<CANDIDATE_INDEX>...` e.g.. `assign i=1 c=4`| Candidates added to interview: [Project Manager [Bernice Yu] 2021-10-20 15:00 - 16:00 PENDING]: <br> 1. David Li |
+| **Edit a candidate** | `edit_c <INDEX> [name=<NAME>] [email=<EMAIL>] [phone=<PHONE_NUMBER>] [address=<ADDRESS>] [status=<STATUS>] [tag=<TAG>]... [position=<POSITION>]...` <br> e.g., `edit_c 3 phone=98602125 email=bryanseah@gmail.com` | Edited Candidate: Bryan Seah; Phone: 98602125; Email: bryanseah@gmail.com; Address: 311, Clementi Ave 2, #02-25 Remark: 20 years of experience Status: SCHEDULED; Positions: [Project Manager] |
+| **Find candidates** | `find_c [name=<NAME>]... [email=<EMAIL>]... [phone=<PHONE_NUMBER>]... [address=<ADDRESS>]... [status=<STATUS>]... [tag=<TAG>]... [position=<POSITION>]...` <br> e.g., `find_c name=Alex tag=recommended priority` | Candidates Found
+| **Add an interview** | `add_i position=<POSITION> [c=<INDEX>]... date=DATE time=TIME duration=DURATION [interviewed=STATUS]` <br> e.g., `add_i position=Accountant c=1 2 date=18/10/2021 time=1400 duration=120 interviewed=pending` | New interview added: [Accountant [Bernice Yu, David Li] 18 Oct 2021 14:00 - 16:00 PENDING] |
+| **Delete an interview** | `delete_i <INDEX>` e.g., `delete_i 1` | Deleted Interview: [Accountant [Bernice Yu, David Li] 18 Oct 2021 14:00 - 16:00 PENDING] |
+| **Edit an interview** | `edit_i <INDEX> [position=POSITION]... [c=<INDEX>]... [date=DATE]... [time=TIME]... [duration=DURATION]... [interviewed=STATUS]...` e.g., `edit_i 2 c=1 2 date=21/10/2021 time=1400` | Edited Interview: [Data Analyst [Jenny Lim, Max Tan] 21 Oct 2021 14:00 - 16:00 PENDING] |
+| **List all interviews** | `list_i` | Listed all interviews <br> 1. [Accountant [Bernice Yu, David Li] 18 Oct 2021 14:00 - 16:00 PENDING] <br> 2. [Project Manager [Bernice Yu] 20 Oct 2021 15:00 - 16:00 PENDING] |
+| **Unassign candidates** | `unassign i=<INTERVIEW_INDEX> c=<CANDIDATE_INDEX>...` e.g., `unassign i=1 c=4`| Candidates removed from interview: [Project Manager [Bernice Yu] 20 Oct 2021 15:00 - 16:00 PENDING]: <br> 1. David Li |
```

Comment:
I think for issue #177, we discussed that it a feedback error. Because Interview card is in YYYY-MM-DD. Only the feedback is in DD MM YYY

Reply:
So the correct format in UG should still be YYYY-MM-DD, and the feedback in the application should be changed to be the same? If so, then I will revert the changes.

Reply:
I think might be better to leave as it is (and maybe change interview card accordingly), because otherwise input is DD/MM/YYYY but display is YYYY-MM-DD. On the other hand 18 Oct 2021 is closer to user's input, but of course can change it to 18-10-2021 or something else entirely also

Reply:
I also think displaying DD MM YYYY for interview card honestly. I feel ppl are more inclined towards DD MM YYYY than YYYY-MM-DD.
So I think can merge, but create an issue to update the interview card to show DD MM YYYY.

Reply:
Sure, I'll merge it now since it looks good otherwise